app/dashboard/templates/dashboard/setting.html

```diff
@@ -183,22 +181,22 @@
             Manage Subscription
           </a>
         {% elif manual_sub %}
-          You are on the Premium plan. The plan ends {{ manual_sub.end_at | dt }}
+          You are on the Premium plan which expires {{ manual_sub.end_at | dt }}
           ({{ manual_sub.end_at.format("YYYY-MM-DD") }}).
           {% if manual_sub.is_giveaway %}
             <br>
-            To support SimpleLogin it's possible to change to a paid plan. <br>
+            To gain additional features and support Simple Login you can upgrade to a Premium plan. <br>
             <a href="{{ url_for('dashboard.pricing') }}" class="btn btn-sm btn-outline-primary">Upgrade</a>
           {% endif %}
         {% elif current_user.lifetime %}
-          You have the lifetime licence.
+          You have lifetime access to the Premium plan.
           {% if not current_user.paid_lifetime %}
             <br>
-            To support SimpleLogin it's possible to change to a paid plan. <br>
+              To gain additional features and support Simple Login you can upgrade to a Premium plan. <br>
```

Comment:
I think we can replace `Premium` by `Paid` here as these users are already having the premium license.

Reply:
So this message is for people on a monthly premium plan?

Reply:
This message is for users who have the lifetime access *for free* at the beginning :). Some of them have asked if they can contribute back by upgrading to a paid plan so we added this message :)